src/hdtSkyrimSystem.cpp

```
@@ -215,6 +217,21 @@ namespace hdt
 		m_model = model;
 		m_filePath = path;
 
+		// Expand any <pattern> macros before parsing, so the loader sees only ordinary SMP elements.
+		// Shared definitions from the global patterns/ folder are visible alongside the file's own.
+		// Fail closed on a malformed pattern: no physics for this item (the asset validator reports why).
+		PatternOptions patternOpts;
+		patternOpts.libraries = &getGlobalPatternLibraries();
+		PatternExpansion expanded = expandPatterns(loaded, patternOpts);
+		if (!expanded.ok) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)